repos/README.md

```diff
@@ -9,8 +9,7 @@ It consists of the following source code repositories and sub-projects:
 ### Provisioning
 
  * [platform-salt](http://github.com/pndaproject/platform-salt): provisioning logic for creating PNDA
- * [pnda-heat-templates](http://github.com/pndaproject/pnda-heat-templates): cluster templates for creating PNDA with Heat
- * [pnda-cli](http://github.com/pndaproject/pnda-cli): cluster templates for creating PNDA with CloudFormation on AWS
+ * [pnda-cli](http://github.com/pndaproject/pnda-cli): cluster templates for creating PNDA on AWS/OpenStack/Baremetal-Servers
```

Comment:
This would read better as 'orchestration application for creating PNDA on AWS, OpenStack or an existing pre-prepared cluster'